docs/api-handlers.md

```diff
@@ -0,0 +1,77 @@
+# Api Handlers
+
+Saleor Apps are meant to work in serverless environment, where Cloud Functions are the foundations of server-side code.
+
+Currently, Saleor heavily relies on Next.js, but in the future, other platforms will be supported.
+
+## Required handlers
+
+Saleor requires 2 endpoints to be available for a standalone app:
+
+- Manifest endpoint - Saleor asks for App information like name and permissions
+- Register endpoint - Saleor uses it to exchange authorization data
+
+TODO - any other endpoints?
```

Comment:
Also the [appUrl](https://docs.saleor.io/docs/3.x/developer/extending/apps/manifest) which is used to display a view in the dashboard when app is opened. Not 100% sure if Saleor allow to create app without it 🤔

Reply:
But appUrl is not endpoint, right? Its set by the factory itself